src/components/single-line-diagram.js

```diff
@@ -45,7 +45,7 @@ const SingleLineDiagram = (props) => {
             var bbox = svg.getBBox();
             svg.setAttribute("viewBox", 0 + " " + 0 + " " + (bbox.width + 20) + " " + (bbox.height + 20));
         }
-    }, []);
+    }, [props]);
```

Comment:
You could be more precise by just adding properties (just property.svg?) that need to reapply the hack

Reply:
ok